### ISSUE TITLE
As a student I want to search and find an institution to attend

### ISSUE THREAD

**Issue by hilabit** [dci-python-backend-assignments/tdd-class-social#1]
As a student I want to be able to search and find institutions by location (address), name of institution, research / educational field.

**Comment by mathiasbrito-dci** [comment]
I think it is fare Hugo, I'll update the models task so that it contains the institution model.

**Comment by hilabit** [comment]
What I've accomplished so far: 
- Installing dependencies 
- Writing first test for positive test case (200 if institution exists) 
- Added properties to class Institution: 
    city, country 
   
What I plan to accomplish today: 
Making the test run and adding a negative test as well as 2 more other tests
What's blocking me: 
1. I get an error "import User" but I can't trace to where it's missing an import. We were working in a group of three and no one was able to point it out. I contacted @mathiasbrito-dci on Thursday for assistance on it so hopefully today we'd be able to go over it together 
2. Need approval for making change in class Institution

**Comment by mathiasbrito-dci** [comment]
Yes, You do have permission to change the institution class. I will check the error with you later.

**Comment by hilabit** [comment]
**What I've accomplished so far:**
- completed two more tests 
**What I plan to accomplish in the next days:**
writing more tests and controllers 

**What's blocking me:**
knowledge but I'm working on it :)

**Comment by mathiasbrito-dci** [comment]
Nice Hila! what knowledge exactly, let us know :)

**Comment by mathiasbrito-dci** [comment]
regarding #21, please add the role to the user as an Optional field... 

`role: Optional[str]`

As an optional field at first, it will not break other's code.
